es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(area, perimeter, width) {
+    super(area, perimeter);
```

Comment:
Utilizing your `width`, you need to calculate `area` & `perimeter`, and that's what you'll pass into `super`